weaviate_cli/commands/create.py

```diff
@@ -283,6 +285,11 @@ def create_backup_cli(ctx, backend, backup_id, include, exclude, wait, cpu_for_b
     default=CreateDataDefaults.vector_dimensions,
     help="Number of vector dimensions to be used when the data is randomized.",
 )
+@click.option(
+    "--uuid",
+    default=None,
+    help="UUID of the object to be used when the data is randomized. The limit will have to be set to 1.",
```

Comment:
Add a note saying that it requires passing --randomize too.

Reply:
Done